go/libkb/api.go

```diff
@@ -189,7 +189,13 @@ func doRequestShared(api Requester, arg APIArg, req *http.Request, wantJSONRes b
 		return
 	}
 
+	dbg := func(s string) {
```

Comment:
Ok, I guess we'll see the timestamps between API GET request and these new ones.